README.md

```diff
@@ -46,9 +46,9 @@ docker run --gpus '"device=0"' -it \
 ### How to Use
 
 ```python
-from deepcell_spots.applications.spot_detection import SpotDetection
+from deepcell_spots.applications.polaris import Polaris
```

Comment:
Let's update the example to use the shortcut import:

`from deepcell_spots.applications import Polaris`